src/pages-and-resources/discussions/data/api.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/prefer-default-export */
+export function getDiscussionApps() {
+  return Promise.resolve({
+    features: [
+      {
+        id: 'lti',
```

Comment:
These IDs are placeholders and we still need to answer the question (long term) about whether they're unique and if there's any namespacing on them.  i.e., `discussions.lti` or something.